Sources/SwiftDocC/Infrastructure/Workspace/DefaultAvailability.swift

```diff
@@ -36,32 +44,82 @@ public struct DefaultAvailability: Codable, Equatable {
         enum CodingKeys: String, CodingKey {
             case platformName = "name"
             case platformVersion = "version"
+            case unavailable = "unavailable"
```

Comment:
minor: It's not necessary to specify the raw value if it's the same as the case name
```suggestion
            case unavailable
```

Reply:
Fixed here https://github.com/apple/swift-docc/pull/807/commits/e4a747366cb1985e176f33eabbed1766c591ab35